frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -16,7 +16,22 @@ const en: Translations = {
         PRIVATE: 'Private service',
         MUNICIPAL_SCHOOL: 'Municipal',
         PRIVATE_SERVICE_VOUCHER: 'Private service (service voucher)'
+      },
+      careTypes: {
+        CLUB: 'Kerho',
```

Comment:
Should we make ticket about translations so that the are double checked at some point?